src/app/components/LocationCard/LocationCard.tsx

```diff
@@ -3,24 +3,32 @@ import styles from './LocationCard.module.css';
 
 export type LocationCardProps = {
   locationName: string | undefined;
-  degree: number;
-  icon: string;
-  description: string;
+  degree: number | undefined;
+  icon: string | undefined;
+  wind: number | null | undefined;
+  description: string | undefined;
```

Comment:
These should never be undefined. You need to check if these are undefined before rendering the component